event_dispatcher.rst

```diff
@@ -129,6 +129,31 @@ listener class:
     internal Symfony listeners usually range from ``-255`` to ``255`` but your
     own listeners can use any positive or negative integer.
 
+
+.. tip::
+
+    When implementing both the ``\Symfony\Component\EventDispatcher\EventListenerInterface``
+    interface, and the ``__invoke`` method with a type-hinted arguments, Symfony
+    uses that type-hint to define the event name.
+    If :ref:`autoconfigure <services-autoconfigure>` is enabled. You neither
+    don't have to manually define the ``kernel.event_listner`` tag.
+    Registering the previous listner could be simplify to:
+
+    .. code-block:: php
+        // src/EventListener/ExceptionListener.php
+        namespace App\EventListener;
+
+        use Symfony\Component\HttpKernel\Event\GetResponseForExceptionEvent;
+
+        class ProcessException implements EventSubscriberInterface
+        {
+            public static function __invoke(GetResponseForExceptionEvent $event)
```

Comment:
If I remember correct, the GetResponseForExceptionEvent was renamed recently by @nicolas-grekas 🤔 and we should reflect that changes

Reply:
fixed. Thank you @OskarStark for that kind review

Reply:
You are very welcome, will have a look tomorrow again, I am currently on a phone 📱